### ISSUE TITLE
TS0601_dimmer_5 switch type attribute not working

### ISSUE THREAD

**Issue by samvel-galstyan** [Koenkk/zigbee2mqtt#23439]
### What happened?

When I am changing the attribute "Switch Type" (Momentary, Toggle, State) nothing happens.

### What did you expect to happen?

I want to set switch mode to toggle.

### How to reproduce it (minimal and precise)

add Moes dimmer (zm-105-M) to Z2M and go to settings. Change the switch type to toggle, state or momentary. Then try to use dumb(rocker) switch. It will act as momentary (push) switch.

### Zigbee2MQTT version

1.39.0-1

### Adapter firmware version

1.42

### Adapter

ZBDongle-E with ezsp

### Setup

Add on on HAOS

**Comment by samvel-galstyan** [comment]
@Koenkk can you please help me?

**Comment by lichtzeichner** [comment]
I have two of these dimmers and I also stumbled upon this issue. I checked the manual, this module is **not** capable of changing the switch type. It expects to be connected to a "rocker switch" (I think in zigbee terms this is a "momentary switch").

See also the manual: https://drive.google.com/file/d/1QHva-X5YcHzhNy_2Jrg2I1MmQKdUKSBU/view

Can we turn this into a bugreport to update the device database to remove the setting? Or should I file a new issue?

**Comment by Pierre-Lavoine** [comment]
Hi there,

I’m going crazy with this module:
https://moeshouse.com/products/zigbee-smart-dimmer-switch-module-timer?variant=47134032691515

Setup:

Zigbee2MQTT (Docker) + Home Assistant (Docker)

Device is MOES ZM-105-M-MS (1 gang)

Detected as: TS0601 / _TZE200_dcnsggvz

Z2M model: TS0601_dimmer_5

Issue #1 — rocker switch behavior is erratic
I’m using a physical rocker (toggle) switch on the input.
Even when I set the device to “toggle/state” (switch_type), the behavior is unpredictable:

sometimes the light turns OFF as expected,

sometimes it starts dimming up/down instead,

sometimes brightness drops to 0% and then jumps back to 1% for no apparent reason.

It feels like the device interprets the rocker action as short/long presses randomly, or ignores the switch_type setting.

Issue #2 — excessive LQI traffic (“spamming” the Zigbee network)
This device publishes LQI extremely frequently (at least ~4 messages per second), which unnecessarily increases Zigbee traffic.

Questions:

Is rocker (toggle) switching actually supported by this device/firmware, or is it effectively “momentary only” regardless of switch_type?

Is there a way to reduce/disable the frequent LQI reporting (or any known workaround)?

Thanks!

**Comment by DJm00n** [comment]
>Is rocker (toggle) switching actually supported by this device/firmware, or is it effectively “momentary only” regardless of switch_type?

Instruction that comes with module says that it "can only be used with reset switch".

https://drive.google.com/file/d/1QHva-X5YcHzhNy_2Jrg2I1MmQKdUKSBU/view

<img width="960" height="1280" alt="Image" src="https://github.com/user-attachments/assets/54ff81db-f965-4db0-a1d1-d1857030130e" />

**Comment by Pierre-Lavoine** [comment]
I'm sorry @DJm00n  but this is not clear, since the wiring diagram clearly show a rocker switch and they also mention it, in option b for reseting the module. ¯\_(ツ)_/¯

<img width="960" height="1280" alt="Image" src="https://github.com/user-attachments/assets/f8e572ec-8047-4889-b444-20c3ba2387b3" />

![Image](https://github.com/user-attachments/assets/4d2f6792-e724-4c54-901b-f1c876a22fec)

**Comment by DJm00n** [comment]
@Pierre-Lavoine yes, its a bit misleading. I think you can ask vendor for clarification.

[This page](https://moeshouse.com/products/zigbee-smart-dimmer-switch-module-timer?variant=47134032691515) says:
> Only self-reset switch available.
